ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -134,32 +136,45 @@ def _router_bgp_vrfs(self) -> dict | None:
                     raise AristaAvdMissingVariableError(f"'vrf_id' or 'vrf_vni' for VRF '{vrf_name}")
 
                 leaf_overlay_rt = f"{self._rt_admin_subfield or bgp_vrf_id}:{bgp_vrf_id}"
-                route_targets = {
-                    "import": {af: [leaf_overlay_rt] for af in vrf_address_families},
-                    "export": {af: [leaf_overlay_rt] for af in vrf_address_families},
-                }
+
+                route_targets = {"import": [], "export": []}
+
+                for af in vrf_address_families:
+                    route_target = {"address_family": af, "route_targets": [leaf_overlay_rt]}
+                    for key, value in route_targets.items():
+                        if route_target not in value:
+                            value.append({"address_family": af, "route_targets": [leaf_overlay_rt]})
```

Comment:
```suggestion
                            value.append(route_target)
```

Reply:
I did this, but getting diff if using the variable - 
```--- a/ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/DC1-BL1A.yml
+++ b/ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/DC1-BL1A.yml
@@ -120,6 +120,7 @@ router_bgp:
         route_targets:
         - '65104:14'
         - 65000:456
+        - 65000:789
       - address_family: vpn-ipv4
         route_targets:
         - 65000:123
@@ -127,6 +128,7 @@ router_bgp:
       - address_family: evpn
         route_targets:
         - '65104:14'
+        - 65000:456
         - 65000:789
       - address_family: vpn-ipv4
         route_targets:
```

Reply:
Understood. Keep it as is.